### PR TITLE
Update Classic bootnodes

### LINKED ISSUE #43: Classic bootnodes are unreachable
Rel #28 .

It seems all of the default Classic bootnodes are unreachable.

The logs below use [a custom-modified ./cmd/bootnode](https://github.com/whilei/multi-geth/tree/fix/bootnodes) program which attempts to validate [the default `Classic` bootnodes](https://github.com/ethoxy/multi-geth/blob/master/params/bootnodes.go#L50-L57). The only one that succeeds is one of my nodes, which I added to the list for comparison.

The reason this is probably causing #28 is because changes to the p2p/ package lately bumped the node db version from 7 to 8 (b/c new/different storage keys) and so per-node "ad hoc" lists of peers aggregated as nodes ran normally are now invalidated, causing peers to fail to find any adequate peers w/o having any adequate bootnodes available.

TODO: Get some new bootnodes!

```
$ ./build/bin/bootnode --bootnodes=classic --nodekey ~/.ethereum-classic/mainnet/nodekey -verbosity 5                                                    
enode://1430a7e07fe0170e5e233edc374ffd375ae0f160eb8eb44a2f0a5f9b50a124b5ef29ae90a427c2061c6d851cbe2c806877e70e69a2bfce92ba788d44c4f8f5cc@127.0.0.1:0?disc
port=30301                                                                                                                                               
Note: you're using cmd/bootnode, a developer tool.                                                                                                       
We recommend using a regular node as bootstrap node for production deployments.                                                                          
using chain bootnodes: classic                                                                                                                           
TRACE[02-19|16:26:44.999] Found seed node in database              id=1969d3adf6797e93 addr=159.203.56.33:30303 age=430726h26m44.999332003s              
INFO [02-19|16:26:44.999] New local node record                    seq=1 id=945054002fc4a318 ip=<nil> udp=0 tcp=0                                        
TRACE[02-19|16:26:44.999] Found seed node in database              id=7443237b72d4b4c6 addr=174.112.32.157:30303 age=430726h26m44.999809079s             
TRACE[02-19|16:26:44.999] Found seed node in database              id=08cf9ce89840d753 addr=52.206.67.235:30303  age=430726h26m44.999821594s             
TRACE[02-19|16:26:44.999] Found seed node in database              id=3064061e8cb4d357 addr=162.243.55.45:30303  age=430726h26m44.999831614s             
TRACE[02-19|16:26:44.999] Found seed node in database              id=ff070c68559f5406 addr=104.155.176.151:30303 age=430726h26m44.999840782s            
TRACE[02-19|16:26:44.999] Found seed node in database              id=a325d4e8dc3bfb45 addr=104.154.136.117:30303 age=430726h26m44.999851542s            
TRACE[02-19|16:26:44.999] Found seed node in database              id=b822f0f3d3d19c91 addr=104.198.71.200:30303  age=430726h26m44.999860705s            
TRACE[02-19|16:26:44.999] Found seed node in database              id=8d853f06e0e14f37 addr=144.76.238.49:30303   age=430726h26m44.999869294s            
TRACE[02-19|16:26:44.999] Found seed node in database              id=4404737930844c9f addr=144.76.238.49:30306   age=430726h26m44.999877747s            
TRACE[02-19|16:26:45.000] Found seed node in database              id=1969d3adf6797e93 addr=159.203.56.33:30303   age=430726h26m45.000097315s            
TRACE[02-19|16:26:45.000] Found seed node in database              id=7443237b72d4b4c6 addr=174.112.32.157:30303  age=430726h26m45.000115396s            
TRACE[02-19|16:26:45.000] Found seed node in database              id=08cf9ce89840d753 addr=52.206.67.235:30303   age=430726h26m45.000125017s            
TRACE[02-19|16:26:45.000] Found seed node in database              id=3064061e8cb4d357 addr=162.243.55.45:30303   age=430726h26m45.000133724s            
TRACE[02-19|16:26:45.000] Found seed node in database              id=ff070c68559f5406 addr=104.155.176.151:30303 age=430726h26m45.000142317s            
TRACE[02-19|16:26:45.000] Found seed node in database              id=a325d4e8dc3bfb45 addr=104.154.136.117:30303 age=430726h26m45.000152235s            
TRACE[02-19|16:26:45.000] Found seed node in database              id=b822f0f3d3d19c91 addr=104.198.71.200:30303  age=430726h26m45.000160653s            
TRACE[02-19|16:26:45.000] Found seed node in database              id=8d853f06e0e14f37 addr=144.76.238.49:30303   age=430726h26m45.000168932s            
TRACE[02-19|16:26:45.000] Found seed node in database              id=4404737930844c9f addr=144.76.238.49:30306   age=430726h26m45.000176954s            
TRACE[02-19|16:26:48.280] >> PING/v4                               id=8d853f06e0e14f37 addr=144.76.238.49:30303   err=nil                                
DEBUG[02-19|16:26:48.780] Removed dead node                        b=13 id=8d853f06e0e14f37 ip=144.76.238.49 checks=0                                    
TRACE[02-19|16:26:55.700] >> PING/v4                               id=ff070c68559f5406 addr=104.155.176.151:30303 err=nil                                
DEBUG[02-19|16:26:56.200] Removed dead node                        b=15 id=ff070c68559f5406 ip=104.155.176.151 checks=0                                  
TRACE[02-19|16:27:04.507] >> PING/v4                               id=4404737930844c9f addr=144.76.238.49:30306   err=nil                                
DEBUG[02-19|16:27:05.007] Removed dead node                        b=16 id=4404737930844c9f ip=144.76.238.49   checks=0                                  
TRACE[02-19|16:27:12.808] >> PING/v4                               id=b822f0f3d3d19c91 addr=104.198.71.200:30303  err=nil                                
DEBUG[02-19|16:27:13.308] Removed dead node                        b=14 id=b822f0f3d3d19c91 ip=104.198.71.200  checks=0                                  
TRACE[02-19|16:27:21.163] >> PING/v4                               id=3064061e8cb4d357 addr=162.243.55.45:30303   err=nil
DEBUG[02-19|16:27:21.664] Removed dead node                        b=16 id=3064061e8cb4d357 ip=162.243.55.45   checks=0                                  
TRACE[02-19|16:27:24.818] >> PING/v4                               id=08cf9ce89840d753 addr=52.206.67.235:30303   err=nil
DEBUG[02-19|16:27:25.319] Removed dead node                        b=16 id=08cf9ce89840d753 ip=52.206.67.235   checks=0                                  
TRACE[02-19|16:27:29.451] >> PING/v4                               id=a325d4e8dc3bfb45 addr=104.154.136.117:30303 err=nil
DEBUG[02-19|16:27:29.952] Removed dead node                        b=14 id=a325d4e8dc3bfb45 ip=104.154.136.117 checks=0
TRACE[02-19|16:27:39.266] >> PING/v4                               id=7443237b72d4b4c6 addr=174.112.32.157:30303  err=nil
DEBUG[02-19|16:27:39.766] Removed dead node                        b=16 id=7443237b72d4b4c6 ip=174.112.32.157  checks=0                                  
TRACE[02-19|16:27:45.739] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
TRACE[02-19|16:27:45.770] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil                                
DEBUG[02-19|16:27:45.770] Revalidated node                         b=16 id=1969d3adf6797e93 checks=1                                                     
TRACE[02-19|16:27:54.160] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil                                
TRACE[02-19|16:27:54.193] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil                                
DEBUG[02-19|16:27:54.193] Revalidated node                         b=16 id=1969d3adf6797e93 checks=2
TRACE[02-19|16:28:00.561] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
TRACE[02-19|16:28:00.596] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil                                
DEBUG[02-19|16:28:00.596] Revalidated node                         b=16 id=1969d3adf6797e93 checks=3
TRACE[02-19|16:28:00.772] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil                                
TRACE[02-19|16:28:00.806] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
DEBUG[02-19|16:28:00.806] Revalidated node                         b=16 id=1969d3adf6797e93 checks=4                                                     
TRACE[02-19|16:28:01.576] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil                                
TRACE[02-19|16:28:01.611] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil                                
DEBUG[02-19|16:28:01.611] Revalidated node                         b=16 id=1969d3adf6797e93 checks=5                                                     
TRACE[02-19|16:28:02.947] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil                                
TRACE[02-19|16:28:02.981] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
DEBUG[02-19|16:28:02.982] Revalidated node                         b=16 id=1969d3adf6797e93 checks=6                                                     
TRACE[02-19|16:28:06.523] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil                                
TRACE[02-19|16:28:06.557] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil                                
DEBUG[02-19|16:28:06.557] Revalidated node                         b=16 id=1969d3adf6797e93 checks=7                                                     
TRACE[02-19|16:28:12.457] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil                                
TRACE[02-19|16:28:12.491] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil                                
DEBUG[02-19|16:28:12.491] Revalidated node                         b=16 id=1969d3adf6797e93 checks=8                                                     
TRACE[02-19|16:28:21.687] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil                                
TRACE[02-19|16:28:21.721] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
DEBUG[02-19|16:28:21.722] Revalidated node                         b=16 id=1969d3adf6797e93 checks=9
TRACE[02-19|16:28:26.977] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
TRACE[02-19|16:28:27.009] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
DEBUG[02-19|16:28:27.009] Revalidated node                         b=16 id=1969d3adf6797e93 checks=10
TRACE[02-19|16:28:33.871] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
TRACE[02-19|16:28:33.904] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
DEBUG[02-19|16:28:33.904] Revalidated node                         b=16 id=1969d3adf6797e93 checks=11
TRACE[02-19|16:28:34.451] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
TRACE[02-19|16:28:34.485] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
DEBUG[02-19|16:28:34.485] Revalidated node                         b=16 id=1969d3adf6797e93 checks=12
TRACE[02-19|16:28:43.136] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
TRACE[02-19|16:28:43.171] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
DEBUG[02-19|16:28:43.172] Revalidated node                         b=16 id=1969d3adf6797e93 checks=13
TRACE[02-19|16:28:48.727] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
TRACE[02-19|16:28:48.760] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
DEBUG[02-19|16:28:48.760] Revalidated node                         b=16 id=1969d3adf6797e93 checks=14
TRACE[02-19|16:28:58.372] >> PING/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
TRACE[02-19|16:28:58.406] << PONG/v4                               id=1969d3adf6797e93 addr=159.203.56.33:30303   err=nil
DEBUG[02-19|16:28:58.406] Revalidated node                         b=16 id=1969d3adf6797e93 checks=15

```

### PR DESCRIPTION
**Pull request by whilei**
Fixes #43 .

Hear ye, hear ye! If ye have some bootnodes ye be willin' an able to contribute to the noble cause, with due respects and by expeditious graces please comment or suggest 'em in this here public forum, and by the will of the mighty Immuteous we may yet keep ourselves a bootnodelicious chain. 

Bootnodes don't have to be special software or anything; in fact it's recommended to use normal full nodes, so long as their ports be open and their aliveness usual.

To find out the `enode` for a client, you can use the `admin.nodeInfo.enode` data, either via RPC (`admin_nodeInfo`) or the use the JS console with something like:

```
$ geth attach
> admin.nodeInfo.enode
```

... if there's a `[::]` in your enode value, just replace it with your actual IP, which you can get with something like `curl icanhazip.com`.

If you've got a running multigeth node already, here's a one-liner (assuming you're not using a custom data dir or anything):

```sh
$ ./build/bin/geth --exec 'admin.nodeInfo.enode' attach
"enode://7188843fa6709100d20a3e3411cd00698bed6901359ef81260d7adc3935d58e5ee28b15d247411cacff0019784a614376a26e7715686baf84aab6dac9c628a3a@24.207.212.9:30303"
# Voila!
```

### CODE REVIEW

**Comment by whilei** [comment]
@soc1c :)?

File: params/bootnodes.go [hunk 1]
```diff
@@ -47,14 +47,7 @@ var EllaismBootnodes = []string{
...
-	"enode://5fbfb426fbb46f8b8c1bd3dd140f5b511da558cd37d60844b525909ab82e13a25ee722293c829e52cb65c2305b1637fa9a2ea4d6634a224d5f400bfe244ac0de@162.243.55.45:30303",
-	"enode://42d8f29d1db5f4b2947cd5c3d76c6d0d3697e6b9b3430c3d41e46b4bb77655433aeedc25d4b4ea9d8214b6a43008ba67199374a9b53633301bca0cd20c6928ab@104.155.176.151:30303",
-	"enode://814920f1ec9510aa9ea1c8f79d8b6e6a462045f09caa2ae4055b0f34f7416fca6facd3dd45f1cf1673c0209e0503f02776b8ff94020e98b6679a0dc561b4eba0@104.154.136.117:30303",
-	"enode://72e445f4e89c0f476d404bc40478b0df83a5b500d2d2e850e08eb1af0cd464ab86db6160d0fde64bd77d5f0d33507ae19035671b3c74fec126d6e28787669740@104.198.71.200:30303",
-	"enode://39abab9d2a41f53298c0c9dc6bbca57b0840c3ba9dccf42aa27316addc1b7e56ade32a0a9f7f52d6c5db4fe74d8824bcedfeaecf1a4e533cacb71cf8100a9442@144.76.238.49:30303",
-	"enode://f50e675a34f471af2438b921914b5f06499c7438f3146f6b8936f1faeb50b8a91d0d0c24fb05a66f05865cd58c24da3e664d0def806172ddd0d4c5bdbf37747e@144.76.238.49:30306",
+	"enode://158ac5a4817265d0d8b977660b3dbe9abee5694ed212f7091cbf784ddf47623ed015e1cb54594d10c1c46118747ddabe86ebf569cf24ae91f2daa0f1adaae390@159.203.56.33:30303", // @whilei
```
Comment (meowsbits) [inline params/bootnodes.go:50]:
```suggestion
	"enode://158ac5a4817265d0d8b977660b3dbe9abee5694ed212f7091cbf784ddf47623ed015e1cb54594d10c1c46118747ddabe86ebf569cf24ae91f2daa0f1adaae390@159.203.56.33:30303", // @whilei

      // etclabs
      "enode://66498ac935f3f54d873de4719bf2d6d61e0c74dd173b547531325bcef331480f9bedece91099810971c8567eeb1ae9f6954b013c47c6dc51355bbbbae65a8c16@54.148.165.1:30303",
      "enode://73e74ce7426a17aa2d8b5bb64d796ec7dc4dcee2af9bbdd4434394d1e4e52e650b9e39202435fca29ce65c242fd6e59b93ed2cf37f04b645fb23e306273816ad@54.148.165.1:30304",
```

Some nodes from @etclabscore

**Comment by soc1c** [comment]
Parity Ethereum Warp Bootnode

```
enode://8809dadb6e6145bbbce832f8e3cb54f23c84db1c4957dac04be26157a16eceb91bb2567e3a2eb794bf8e7df937fd2de46d4214416d3c199f05d3b6fa76a5ef6d@51.141.115.3:30303
```

File: params/bootnodes.go (hunk 1 above)
Comment (soc1c) [inline params/bootnodes.go:50]:
```suggestion
	"enode://158ac5a4817265d0d8b977660b3dbe9abee5694ed212f7091cbf784ddf47623ed015e1cb54594d10c1c46118747ddabe86ebf569cf24ae91f2daa0f1adaae390@159.203.56.33:30303", // @whilei
	
	// parity
	"enode://8809dadb6e6145bbbce832f8e3cb54f23c84db1c4957dac04be26157a16eceb91bb2567e3a2eb794bf8e7df937fd2de46d4214416d3c199f05d3b6fa76a5ef6d@51.141.115.3:30303",  
```

Comment (sorpaas) [reply params/bootnodes.go:50]: Added

Comment (sorpaas) [reply params/bootnodes.go:50]: Added

**Pull request merged**